src/storage/remote-db-adapter.js

```diff
@@ -46,15 +47,21 @@ export default class RemoteDBAdapter {
    * @param {WordItem} data
    * @return {Boolean} - successful/failed result
    */
-  async update(data) {
+  async update(data, skipSerialize = false) {
     try {
       let url = this.dbDriver.storageMap.put.url(data)
-      let content = this.dbDriver.storageMap.put.serialize(data)
+      let content
+      if (skipSerialize) {
```

Comment:
please add a comment to explain why we skip serialization

Reply:
Added